org.virt_manager.virt-viewer.yml

```diff
@@ -0,0 +1,186 @@
+app-id: org.virt_manager.virt-viewer
+runtime: org.gnome.Platform
+runtime-version: "43"
+sdk: org.gnome.Sdk
+command: remote-viewer
+rename-icon: virt-viewer
+rename-desktop-file: remote-viewer.desktop
+rename-appdata-file: remote-viewer.appdata.xml
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=ssh-auth
+  - --device=all
+  - --share=ipc
+  - --share=network
+cleanup:
+  - "/include"
+  - "/sbin"
+  - "/lib/pkgconfig"
+  - "/share/man"
+  - "/share/gtk-doc"
+  - "/share/gir-*"
+  - "/share/pkgconfig"
+  - "*.la"
+  - "*.a"
+modules:
+  - name: lz4
+    build-options:
+      env:
+        PREFIX: /app
+    buildsystem: simple
+    build-commands:
+      - make -j $FLATPAK_BUILDER_N_JOBS
+      - make install
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: https://github.com/lz4/lz4/archive/v1.9.4.tar.gz
+        sha256: 0b0e3aa07c8c063ddf40b082bdf7e37a1562bda40a0ff5272957f3e987e0e54b
+  - name: gtk-vnc
+    buildsystem: meson
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/gtk-vnc/1.3/gtk-vnc-1.3.1.tar.xz
+        sha256: 512763ac4e0559d0158b6682ca5dd1a3bd633f082f5e4349d7158e6b5f80f1ce
+  - name: libvirt-glib
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://libvirt.org/sources/glib/libvirt-glib-4.0.0.tar.xz
+        sha256: 8423f7069daa476307321d1c11e2ecc285340cd32ca9fc05207762843edeacbd
+    modules:
+      - name: libnl
+        config-opts:
+        - "--enable-cli=no"
+        sources:
+          - type: archive
+            url: >-
+              https://github.com/thom311/libnl/releases/download/libnl3_5_0/libnl-3.5.0.tar.gz
+            sha256: 352133ec9545da76f77e70ccb48c9d7e5324d67f6474744647a7ed382b5e05fa
+      - name: portablexdr
+        buildsystem: autotools
+        config-opts:
+          - "--disable-static"
+        sources:
+          - type: archive
+            url: >-
+              https://people.redhat.com/~rjones/portablexdr/files/portablexdr-4.9.1.tar.gz
+            sha256: 5cf4bdd153cf4d44eaf10b725f451d0cfadc070b4b9a9ccfb64094b8f78de72c
+          - type: script
+            dest-filename: autogen.sh
+            commands:
+              - AUTOMAKE="automake --foreign" autoreconf -vfi
+          - type: shell
+            commands:
+              - cp -f /usr/share/gnu-config/config.sub .
+              - cp -f /usr/share/gnu-config/config.guess .
```

Comment:
I guess you're including this to get the 'rpcgen' binary ?   I'd generally suggest not using portablexdr at all, as its author only really created it to fill a gap on mingw. rpcgen would be the recommended package  to provide an rpcgen binary needed by libvirt https://github.com/thkukuk/rpcsvc-proto

Reply:
...in fact you have rpcsv-proto later already, so portablexdr should be totally redundant.

Reply:
This too got from https://github.com/flathub/org.gnome.Boxes/blob/master/org.gnome.Boxes.json